src/handlers/scan-contributions.ts

```diff
@@ -0,0 +1,132 @@
+
+  const repo = payload.repository.name;
+  const issueNumber = payload.issue.number;
+  const owner = payload.repository.owner.login;
+  const body = payload.comment.body;
+
+  if (!body.match(/scan-contributions/i)) {
+    return;
+  }
+
+  logger.info("Scanning Events!");
+
+  try {
+    const contributors = await octokit.repos.listContributors({
+      repo,
+      owner,
+    });
+
+    contributors.data.forEach((contributor) => {
+      if (contributor.login) {
+        store[contributor.login] = {};
+      }
+    });
+
+    const issueTimelineEvents = await octokit.paginate(octokit.issues.listEventsForTimeline, { owner, repo, issue_number: issueNumber });
+    const issueEvents = await octokit.paginate(octokit.issues.listEvents, { owner, repo, issue_number: issueNumber });
+    const pullRequestReviewsEvents = await octokit.paginate(octokit.pulls.listReviews, { owner, repo, pull_number: issueNumber });
+    const issueReactionEvents = await octokit.paginate(octokit.reactions.listForIssue, { owner, repo, issue_number: issueNumber });
+    // const issueCommentsReactionEvents = await Promise.all(
+    //   (await octokit.paginate(octokit.issues.listComments, { owner, repo, issue_number: issueNumber })).map((comment) =>
+    //     octokit.paginate(octokit.reactions.listForIssueComment, { owner, repo, comment_id: comment.id })
+    //   )
+    // );
+    // const issueReviewCommentsReactionEvents = await Promise.all(
+    //   (await octokit.paginate(octokit.pulls.listReviews, { owner, repo, pull_number: issueNumber })).map((comment) =>
+    //     octokit.paginate(octokit.reactions.listForIssueComment, { owner, repo, comment_id: comment.id })
+    //   )
+    // );
+
+    issueTimelineEvents.forEach((ev) => {
+      if ("actor" in ev && ev.actor && store[ev.actor.login]) {
+        if (!store[ev.actor.login][ev.event]) store[ev.actor.login][ev.event] = 1;
+        else store[ev.actor.login][ev.event] += 1;
+      }
+    });
+
+    issueEvents.forEach((ev) => {
+      if (ev.actor && store[ev.actor.login] && !issueTimelineEvents.map((te) => te.event).includes(ev.event)) {
+        if (!store[ev.actor.login][ev.event]) store[ev.actor.login][ev.event] = 1;
+        else store[ev.actor.login][ev.event] += 1;
+      }
+    });
+
+    pullRequestReviewsEvents.forEach((ev) => {
+      const eventName = "review[".concat(ev.state.toLowerCase()).concat("]");
+      if (ev.user && store[ev.user.login]) {
+        if (!store[ev.user.login][eventName]) store[ev.user.login][eventName] = 1;
+        else store[ev.user.login][eventName] += 1;
+      }
+    });
+
+    issueReactionEvents.forEach((ev) => {
+      const eventName = "reaction_" + ev.content;
+      if (ev.user && store[ev.user.login]) {
+        if (!store[ev.user.login][eventName]) store[ev.user.login][eventName] = 1;
+        else store[ev.user.login][eventName] += 1;
+      }
+    });
+
+    // issueCommentsReactionEvents.forEach((event) => {
```

Comment:
You should remove unused code

Reply:
Those codes could be used later to capture more relevant events. I commented them out because they were causing a "too many subrequests error" when the command was invoked.

Reply:
If the code doesn't work then it needs to be reimplemented